### PR TITLE
ttiex355p: emulating correctly for bug fix tests

### PR DESCRIPTION
**Pull request by JamesKingWork**
ttiex355p:

- Add minimum to current and voltage range

https://github.com/ISISComputingGroup/IBEX/issues/6397

### CODE REVIEW

File: lewis_emulators/tti355/device.py [hunk 1]
```diff
@@ -21,6 +21,8 @@ def _initialize_data(self):
         self._max_voltage = 35.0
         self._max_current = 5.0
         self.load_resistance = 10
+        self.min_voltage = None
+        self.min_current = None
```
Comment (DominicOram) [inline lewis_emulators/tti355/device.py]: As I said in the other PR, can probably put the sim/max stuff in here rather than None

**Review by DominicOram — CHANGES_REQUESTED** [review]
Good, minor comment in code

File: lewis_emulators/tti355/device.py (hunk 1 above)
Comment (JamesKingWork) [reply lewis_emulators/tti355/device.py]: Utilised sub-classing to provide extra check with the ttiex355p emulator rather than using the tti355

**Review by DominicOram — APPROVED** [review]
Looks good and tests pass, thanks

**Pull request merged**